setup.cfg

```diff
@@ -15,17 +15,17 @@ isort_ignore=
 line_length=80
 indent='    '
 not_skip=__init__.py
-skip=wrapt,setup.py
+skip=wrapt,setup.py,six.py,build
 multi_line_output=0
 known_standard_library=importlib,types,asyncio
 known_django=django
 known_first_party=elasticapm,tests
-known_third_party=pytest,flask,aiohttp,urllib3_mock,webob,memcache,pymongo,boto3,logbook,twisted,celery,zope,urllib3,redis,jinja2,requests,certifi
+known_third_party=pytest,flask,aiohttp,urllib3_mock,webob,memcache,pymongo,boto3,logbook,twisted,celery,zope,urllib3,redis,jinja2,requests,certifi,mock
 default_section=FIRSTPARTY
 sections=FUTURE,STDLIB,DJANGO,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
 
 
 [flake8]
-exclude=elasticapm/utils/wrapt
+exclude=elasticapm/utils/wrapt,elasticapm/utils/six.py,build
```

Comment:
@beniwohli Is `six` still needed here?

Reply:
nope, can be removed (but this branch probably needs a rebase to get the six-removal)